### PR TITLE
Updating fabricbot area pods

### PR DESCRIPTION
**Pull request by joperezr**
Applying https://github.com/dotnet/fabricbot-config/pull/62

cc: @jeffhandley @ericstj

### CODE REVIEW

**Review by jeffhandley — DISMISSED** [review]
Please also update `area-pods.json` (which is what drives issuesof.net). I don't think there are any changes necessary to `area-pods.md`, but it's worth a double-check.

**Comment by joperezr** [comment]
Just to confirm, I checked `area-owners.md` and it indeed doesn't seem to need an update.

**Review by ericstj — APPROVED** [review]

**Comment by joperezr** [comment]
Just to confirm, I have now renamed both the PR and Issue Triage boards to reflect the changes here so that automation continues to work.

File: docs/area-owners.json [hunk 1]
```diff
@@ -616,12 +616,11 @@
     },
     {
       "lead": "ericstj",
-      "pod": "buyaa-jose-steve",
+      "pod": "buya-steve",
```
Comment (danmoseley) [inline docs/area-owners.json:619]: should this have two a's?

**Pull request merged**